README.md

```diff
@@ -28,19 +28,18 @@ yarn build          # Run dev
 yarn serve          # Serve the prod build
 ```
 
-## Configure
+## Contributing
 
-### Algolia
+We welcome contributions! Follow this [link](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/src/main/asciidoc/appendix-contributing.adoc) for more information on how to contribute.
```

Comment:
We could add a boilerplate to highlight that all the contributions are as simple as markdown files. Perhaps also point to the `./content/documentation` folder in a link, so folks can quickly get to it and relate to the website.

Reply:
Also, I'm not sure pointing to SCDF's contribution guide would be useful for this repo, though. The content is kind of different in some ways.